cfgmgr/intfmgr.cpp

```diff
@@ -173,7 +196,11 @@ bool IntfMgr::doIntfAddrTask(const vector<string>& keys,
             return false;
         }
 
-        setIntfIp(alias, "add", ip_prefix.to_string(), ip_prefix.isV4());
+        // Set Interface IP except for lo
+        if (!is_lo)
```

Comment:
Please explain how your code is dealing with multiple IP addresses on the loopback which is a valid config and must continue to work.

Reply:
What's the issue with multiple IPs?
It's just different keys in APP DB

Reply:
Today the loopback in sonic has more than one address. So with your changes post https://github.com/Azure/sonic-swss/pull/635, will all the loopback addresses make it to the asic and correctly? I'm just asking you to explain and also test.